save3ds_fuse/src/main.rs

```diff
@@ -1420,7 +1430,8 @@ fn main_inner() -> Result<(), Box<dyn std::error::Error>> {
             resource.open_nand_save(id, !read_only)?,
             operation,
             mountpoint,
-        )?
+        )
+        .unwrap()
```

Comment:
There is still unwrap here. I don't want to add comment on every single one of them. Please clean them up in the entire code